### ISSUE TITLE
search without autocompletion

### ISSUE THREAD

**Issue by saubin78** [agroportal/project-management#483]
When in the Concepts view of a thesaurus, I can search for concepts in the "jump to" box but how can I launch a search on, e.g. "pomme" and get a list of **all the concepts** that contain "pomme" in its pref or altLabels? 
I would expect a search functionality similar to the green bar search (on all ontologies) but restricted to the one I am viewing

**Comment by syphax-bouazzouni** [comment]
TODO

- Update the jump to use the same component as in the home page, which will redirect to the search page.

**Comment by Bilelkihal** [comment]
@syphax-bouazzouni 
I think that it would be confusing to redirect the user to the search page when perform a search in this section and click on a result from the drop down list.
Instead, we can improve the current jump_to input to include `altLabels` cause currently it already supports searching by labels and synonyms (I'm not sure if our search api doesn't do the same thing)
![image](https://github.com/user-attachments/assets/aa9d6071-34c3-411c-a61d-564591e17b16)

**Comment by syphax-bouazzouni** [comment]
> I think that it would be confusing to redirect the user to the search page when perform a search in this section and click on a result from the drop-down list.

The idea here is to redirect to the search page, but auto-selecting the current ontology the ontology selector. 
At the same time, this UI component needs to be updated, and no more use the automcoplete.js  file (listed here https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/426)

> Instead, we can improve the current jump_to input to include altLabels cause currently it already supports searching by labels and synonyms (I'm not sure if our search api doesn't do the same thing)

It does already include `altLabels`, as synonyms. It also uses the `search` endpoint and the background. The logic is in the file `ontoportal_autocomplete_controller.js` that we need to be removed also.

**Comment by jonquet** [comment]
IMO, I would not have the Jump to menu bring you back to Search page. I think it would be very confusing. The "Jump to" box is a box to isolate quickly where to go with a selector. Not a search. That's why its called "Jump to" and not "Search". What we can do is to unify the behaviour of this "Jump to" with the other ones on the Property/Instances/Scheme/Collections tabs and rename them all these menus "Filter" and then display in the tree only the terms that contains the string in the filter. 
(because in fact the menus on the other tabs are more "filter" than "search").

In other words, I would : 
- [ ]  either do nothing and leave the Jump to as it is.. and in a sense discard @saubin78 's request
- [ ] unify the Jump to to become a filter (but maybe that need to wait for more work on the tree view)

**Comment by imadbourouche** [comment]
related to: 
- https://github.com/agroportal/project-management/issues/634
